src/main/java/li/cil/tis3d/common/module/DisplayModule.java

```diff
@@ -150,7 +158,7 @@ public void onDisposed() {
 
     @Override
     public void finalize() {
-        LeakDetector.add(glTextureId, nativeImage);
+        //~ LeakDetector.add(glTextureId, nativeImage);
```

Comment:
I reviewed #93, but the issue opener never reported back so I was unsure what to do here. Personally I didn't encounter any crashes when testing the TextureManager unregister code, but that was only within single player. I will reinstate the leak detector code.

Reply:
Taking a closer look at this, I'm wondering why `deleteTexture()` was ever called from `finalize()` in the first place. The call from `onDisposed()` is correct assuming this method is called at least once near the end of the module's lifetime. Since the GC can in theory run on a different thread, `finalize()` is most definitely the wrong place for it.

I believe the `LeakDetector` is superfluous but I will leave its logic untouched since this is technically not related to the port.